10-01 21:15:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "FILE" not found; SQL statement:
DROP TABLE FILE [42102-172]
10-01 21:25:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "C" not found; SQL statement:
insert into file (drive, path, filename, md5) VALUES (C, './test/folder2/','imgp0002.jpg','22d2b296d0b1586af2cb923f1201d2ea') [42122-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:237)
	at org.h2.command.Parser.prepareCommand(Parser.java:219)
	at org.h2.engine.Session.prepareLocal(Session.java:425)
	at org.h2.engine.Session.prepareCommand(Session.java:374)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:167)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:155)
	at com.imageverifier.DB.insert(DB.java:57)
	at com.imageverifier.FileWalker.walk(FileWalker.java:27)
	at com.imageverifier.FileHavester.recursiveHarvest(FileHavester.java:32)
	at com.imageverifier.FileHavester.main(FileHavester.java:21)
10-01 21:39:41 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:655)
	at org.h2.server.web.WebApp.test(WebApp.java:840)
	at org.h2.server.web.WebApp.process(WebApp.java:215)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:138)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 14 more
10-01 21:39:41 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:655)
	at org.h2.server.web.WebApp.test(WebApp.java:840)
	at org.h2.server.web.WebApp.process(WebApp.java:215)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:138)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 14 more
10-01 21:39:42 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:171)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:90)
	at org.h2.Driver.connect(Driver.java:73)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:655)
	at org.h2.server.web.WebApp.test(WebApp.java:840)
	at org.h2.server.web.WebApp.process(WebApp.java:215)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:164)
	at org.h2.server.web.WebThread.process(WebThread.java:138)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-172]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 14 more
